src/app/api/admin/sendAdminNewDeal/route.tsx

```diff
@@ -0,0 +1,17 @@
+import { inngest } from '@/utils/inngest/client'
+
+export async function POST(request: Request) {
+  const body = await request.json()
+  try {
+    const { ids } = await inngest.send({
```

Comment:
This logic doesn't need to live in an API route. It can be called directly from the action. Since the action runs on the server, this code should be used there directly.

Reply:
Good call! I forgot to mention this in the PR, initially when we spoke we discussed using an api route to trigger this flow, wasn't sure if there were other plans for this route as well. I'll remove it.